File: src/lib/engine/crypto-config.ts
import { DEFAULT_RISK_SETTINGS, type RiskSettings } from "./quant";

export const CRYPTO_RISK_SETTINGS: RiskSettings = {
  ...DEFAULT_RISK_SETTINGS,
  atrStopMultiplier: 2,
  minimumRewardRisk: 1.6,
  maxRiskPerTradePercent: 0.5,
};

File: engine/smc/crypto_config.py
"""Crypto-calibrated risk settings (port of crypto-config.ts)."""

from dataclasses import replace

from smc.quant import DEFAULT_RISK_SETTINGS, RiskSettings

CRYPTO_RISK_SETTINGS: RiskSettings = replace(
    DEFAULT_RISK_SETTINGS,
    atr_stop_multiplier=2,
    minimum_reward_risk=1.6,
    max_risk_per_trade_percent=0.5,
)
